lib/contextMenu.js

```diff
@@ -722,8 +754,15 @@ export default function (self) {
         contextPosition.width =
           self.width - self.style.mobileContextMenuMargin * 2 + 'px';
       }
+      if (e.type == 'mousedown') {
+        contextPosition.top += 10;
```

Comment:
It's not clear to me exactly what this `10` represents — can we assign it to a variable, so it has a name? It seems to be an offset of some kind; would it make sense to make it a configurable style option?